src/DependencyInjection/Greg0ireEnumExtension.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace Greg0ire\EnumBundle\DependencyInjection;
+
+use Symfony\Component\Config\FileLocator;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Loader;
+use Symfony\Component\HttpKernel\DependencyInjection\Extension;
+
+
+class Greg0ireEnumExtension extends Extension
+{
+
+    public function load(array $configs, ContainerBuilder $container)
+    {
+        $configuration = new Configuration();
+        $config = $this->processConfiguration($configuration, $configs);
+
+        $loader = new Loader\XmlFileLoader($container, new FileLocator(__DIR__.'/../Resources/config'));
+        $loader->load('formatters.xml');
+
+        $container->setParameter('greg0ire_enum.formatter', $config['formatter']);
```

Comment:
What if `$config['formatter']` is not set ? It crashes. Do you know how I noticed that ? By writing tests!